packages/builder/src/components/common/CodeEditor/CodeEditor.svelte

```diff
@@ -298,27 +280,194 @@
 </div>
 
 <style>
-  .code-editor.handlebars :global(.cm-content) {
-    font-family: var(--font-sans);
+  /* Editor */
```

Comment:
I basically just found it very difficult to reason about what styles were applied where, and what they affected, because some were defined in CSS and some were defined as part of the theme config. There were inconsistencies around which style overode I found the theme config much more awkward to use, both because you need to pass in a bunch of props to ensure styles are generated, and also just the camel case naming and string value syntax of the styles built up.

I found it much easier to have a single source of truth which we can style with proper CSS (and it's also technically more performant, not that that's a consideration here but it's free).

It was also very difficult to actually reason which styles affected what, which is why I've now broken it down into sections and commented appropriately, to try and help people in future be able to make style changes without trying to work out the consequences.